app/services/DynamoArchivedChannelTests.scala

```diff
@@ -0,0 +1,64 @@
+package services
+
+import com.typesafe.scalalogging.StrictLogging
+import models.DynamoErrors._
+import models._
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+import software.amazon.awssdk.services.dynamodb.model._
+import zio.blocking.effectBlocking
+import zio.duration.durationInt
+import zio.stream.ZStream
+import zio.{ZEnv, ZIO}
+
+import scala.jdk.CollectionConverters._
+
+
+class DynamoArchivedChannelTests(stage: String, client: DynamoDbClient) extends StrictLogging {
```

Comment:
The `putAll` and `putAllBatched` are copied from the existing `DynamoChannelTests.scala`. This feature is temporary so it's not worth a refactor